core/src/main/scala/chisel3/dontTouch.scala

```diff
@@ -2,10 +2,11 @@
 
 package chisel3
 
-import chisel3.experimental.{annotate, requireIsHardware, ChiselAnnotation}
+import chisel3.experimental.{annotate, requireIsAnnotatable, ChiselAnnotation}
 import chisel3.properties.Property
 import chisel3.reflect.DataMirror
 import firrtl.transforms.DontTouchAnnotation
+import chisel3.experimental.requireIsHardware
```

Comment:
It seems like this should be combined in the first import.

Reply:
That second one is actually unused, I readded it by accident (for testing), will remove!

Reply:
Fixed!